studio/app/common/dataclass/polar.py

```diff
@@ -33,14 +33,37 @@ def output_path(self) -> OutputPath:
 
     def save_plot(self, output_dir):
         for i in range(len(self.data)):
-            fig = px.line_polar(
-                r=self.data[i],
-                theta=self.columns,
-                direction="counterclockwise",
-                start_angle=0,
-                line_close=True,
+            plt.figure(figsize=(10, 10))
+            ax = plt.subplot(111, polar=True)
+
+            # Convert degrees to radians
+            theta_rad = np.radians(self.columns)
+
+            # Basic plot
+            ax.plot(theta_rad, self.data[i], color="#636EFA", linewidth=1.5)
+
+            # Set direction and orientation to match plotly defaults
+            ax.set_theta_zero_location("E")  # 0° at top
+            ax.set_theta_direction(1)  # counterclockwise
+
+            # Set degree labels
+            ax.set_xticks(np.radians([0, 45, 90, 135, 180, 225, 270, 315]))
+            ax.set_xticklabels(
+                ["0°", "45°", "90°", "135°", "180°", "225°", "270°", "315°"]
             )
-            plot_file = join_filepath([output_dir, f"{self.file_name}_{i}.png"])
-            pio.write_image(fig, plot_file)
+
+            # Close the line
+            if len(theta_rad) > 0:
+                ax.plot(
+                    [theta_rad[-1], theta_rad[0]],
+                    [self.data[i][-1], self.data[i][0]],
+                    color="#636EFA",
+                    linewidth=1.5,
+                )
+
+            plot_file = f"{output_dir}/{self.filename}_{i}_matplotlib.png"
```

Comment:
This part is not working due to a typo.
Please confirm that the process goes through without error.
*As I have pointed out before, the comments seem to be disappearing with force-commit.